test/blackbox-tests/test-cases/bisect-ppx/bisect-exe/dune

```diff
@@ -0,0 +1,9 @@
+(library
```

Comment:
Nowadays we prefer to create files from the `run.t` file directly, i.e. using this syntax:

```
  $ cat >dune<<EOF
  > (library
  >  (name hello)
  > ...
  > EOF
```

except for large files. This makes the test more self-contained and it is easier to follow the logic of the test. Do you mind changing this test to create the files inside `run.t`?

Reply:
Done!

Reply:
Thanks!